pkg/httputil/logging.go

```diff
@@ -53,13 +54,23 @@ func RequestID(r *http.Request) (*http.Request, string) {
 	return r, reqID
 }
 
+func SourceIP(r *http.Request) string {
+	sourceIP, sourcePort, err := net.SplitHostPort(r.RemoteAddr)
+
+	if err != nil {
+		return "Could not get IP address" //not sure of the error handling here
+	}
+	return sourceIP + sourcePort
```

Comment:
Not sure this will be properly formatted anyway, I'd expect needing to add a colon char as a separator. Doesn't matter if we do the above suggestion, wdyt?

Reply:
You're absolutely right.

Reply:
```suggestion
	return sourceIP + ":" + sourcePort
```